projects/core/components/dialog/dialog.style.less

```diff
@@ -2,15 +2,19 @@
 
 :host {
     .text-body-m();
+
+    .dialog-host {
```

Comment:
Let's not move it under `:host`, it increases specificity and there's no reason to do so.

Reply:
done

Reply:
I mean move it outside of `:host { ... }` :) Otherwise it adds `[ng-host-xxxx]` as a parent to the style which increases the specificity.

Reply:
done